tests/authentication/models/test_user.py

```diff
@@ -0,0 +1,31 @@
+from app.authentication.models.user import User
+from tests import BaseTestCase
+
+import faker
+
+
+FAKER = faker.Faker()
+
+
+class UserTestCase(BaseTestCase):
+    """User model TestCase"""
+
+    def setUp(self):
+        super(UserTestCase, self).setUp()
+        self.raw_password = FAKER.password()
+        self.user = User(email=FAKER.email(), password=self.raw_password)
```

Comment:
Why not have only one **setUp method** as defined in **BaseTestCase**. So this can also be moved to **setUp** in BaseTestCase. 
I like this faker library.

Reply:
yeah that would work but remember the `BaseTestCase` will be subclassed by other test cases moving forward. So this means I will be bloating the `setUp` method in the superclass with un-necessary attributes which may not be used by the other test cases